reddit_liveupdate/controllers.py

```diff
@@ -979,6 +979,24 @@ def GET_home(self):
             page_classes=["liveupdate-home"],
         ).render()
 
+    @require_oauth2_scope("read")
+    @api_doc(
+        section=api_section.live,
+        uri="/live/happening-now",
+    )
+    def GET_happening_now(self):
+        """Redirect to the live thread "happening now".
+
+        "happening now" means it's currently promoted on the frontpage.
+        Will 404 if there is not live thread "happening now".
+        """
+        current_thread_id = NamedGlobals.get(HAPPENING_NOW_KEY, None)
+        if current_thread_id:
+            current_thread = LiveUpdateEvent._byID(current_thread_id)
```

Comment:
Quick note: I don't see any admin endpoint to *remove* the thread from happening now. So assuming that's not handled privately/manually it would be an issue, unless I'm missing something.